src/foam/nanos/auth/Group.js

```diff
@@ -118,6 +118,10 @@ foam.CLASS({
       name: 'replyTo',
       value: null
     },
+    {
+      class: 'Long',
+      name: 'softSessionLimit'
```

Comment:
Please document this. regarding how it's used and what values are appropriate and in what units - milliseconds, seconds.  And what values are appropriate like 1 minute.   And how it interacts with Session limits.

Reply:
Documentation added